test/functional/device-spec.js

```diff
@@ -1,16 +1,30 @@
 /* eslint no-undef: "off" */
-const assert = require('assert').strict
+const Assert = require('assert').strict
+const dotenv = require('dotenv')
 const StApi = require('../../lib/api')
+const Client = require('../../lib/platform/client')
+
+dotenv.config()
 // User token for prod env
-const opts = {authToken: '0594f1d2-69e9-4809-a773-5fe7268e3f01'}
+const opts = {authToken: PAT}
```

Comment:
`PAT`: are you intending to use the environment variable here? Like `process.env.PAT`?